laserscan_kinect/config/params.yaml

```diff
@@ -1,24 +1,26 @@
 # Frame id for the output laserscan.
```

Comment:
This file did not need to be changed in this repo, it could easily be added to one of our robot config repos and passed to the launch file